sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/batch_job.py

```diff
@@ -0,0 +1,59 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict
+
+from azure.ai.ml._schema._deployment.batch.batch_job import BatchJobSchema
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+from azure.ai.ml.entities._deployment.batch_job_property import BatchJobProperty
+from azure.ai.ml.entities import SystemData
+from azure.ai.ml._restclient.v2020_09_01_dataplanepreview.models import BatchJobResource as RestBatchJobResource
+
+
+class BatchJob:
+    """Batch Job entity
+
+    :param id: Fully qualified resource ID for the resource. Ex -
+     /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}.
+    :type id: str
+    :param name: The name of the resource.
+    :type name: str
+    :param type: The type of the resource. E.g. "Microsoft.Compute/virtualMachines" or
```

Comment:
What does this actually mean? How do I invoke a vm batch job vs a storage batch job? I'm thinking if this is something we expose to customers or leave out for now.

Reply:
I got that definition from the BatchJobResource API contract, I'm not quite sure what that means. Maybe a better question for Fecund?